issue found on: server/server.go:43:10: importShadow

an issue has been detected in source code: ./server/server.go:43:10: importShadow: shadow of imported from 'github.com/RedHatInsights/ccx-data-pipeline-monitor/config' package 'config'

--- a/server/server.go
+++ b/server/server.go
@@ -38,9 +38,9 @@ type HTTPServer struct {
 }
 
 // New constructs new implementation of Server interface
-func New(config config.ServerConfig) *HTTPServer {
+func New(configuration config.ServerConfig) *HTTPServer {
 	return &HTTPServer{
-		Config: config,
+		Config: configuration,
 	}
 }
 

